homeassistant/components/switch/broadlink.py

```diff
@@ -36,18 +40,27 @@
             'rm2_pro_plus_bl', 'rm_mini_shate']
 SP1_TYPES = ['sp1']
 SP2_TYPES = ['sp2', 'honeywell_sp2', 'sp3', 'spmini2', 'spminiplus']
+MP1_TYPES = ["mp1"]
 
-SWITCH_TYPES = RM_TYPES + SP1_TYPES + SP2_TYPES
+SWITCH_TYPES = RM_TYPES + SP1_TYPES + SP2_TYPES + MP1_TYPES
 
 SWITCH_SCHEMA = vol.Schema({
     vol.Optional(CONF_COMMAND_OFF, default=None): cv.string,
     vol.Optional(CONF_COMMAND_ON, default=None): cv.string,
     vol.Optional(CONF_FRIENDLY_NAME): cv.string,
 })
 
+MP1_SWITCH_SLOT_SCHEMA = vol.Schema({
+    vol.Optional('slot_1', default=None): cv.string,
+    vol.Optional('slot_2', default=None): cv.string,
+    vol.Optional('slot_3', default=None): cv.string,
+    vol.Optional('slot_4', default=None): cv.string
+})
```

Comment:
Do not set default `None`

Reply:
removed. btw, could u explain why we should not set default `None` here?